pyamg/aggregation/adaptive.py

```diff
@@ -113,7 +113,7 @@ def unpack_arg(v):
 def adaptive_sa_solver(A, initial_candidates=None, symmetry='hermitian',
                        pdef=True, num_candidates=1, candidate_iters=5,
                        improvement_iters=0, epsilon=0.1,
-                       max_levels=10, max_coarse=100, aggregate='standard',
```

Comment:
I agree that the max_coarse for all the solvers should be the same (which it wasn't until your change).  

However, I think that max_coarse should be as large as possible.  I did some timings on my laptop and pinv(rand(500,500)) takes about 1s on my laptop.  That's probably too slow.  But for a (200,200) matrix, the timing was about 0.05s.  So, I think changing this to 200 makes sense.  I think 10 is a little small.